src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -64,4 +64,9 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
+
+    @Override
+    public CommandResult execute(CareFlowModel model) throws CommandException {
```

Comment:
Are we keeping both CareFlowModel and Model?

Reply:
I believe Model has been removed, to prevent confusion.